### DISCUSSION TITLE
Service traceid search doesn't work in latest version

### DISCUSSION THREAD

**Discussion by forgottener** [apache/skywalking#12494] [Q&A] [answered]
When I search in the service trace using the specified traceid, I cannot correctly locate the trace link I want to view.
Is my search method wrong or is it due to other reasons?

See the video below.
https://github.com/user-attachments/assets/ce832a4c-5eb9-4d74-a925-b30711fb34bf

**Comment by wu-sheng** [comment]
We may fix in the latest commit. You could recheck.

**Reply by forgottener** [reply]
This issue still exists in banyandb 0.7.0. live demo:[https://demo.skywalking.apache.org/](url)
which milestone will it be fixed？

**Reply by hanahmily** [reply, marked as answer]
We failed to file an issue to track it. I just submit https://github.com/apache/skywalking/issues/12745
